README.md

```diff
@@ -148,6 +148,19 @@ command \
 ```
 This takes inspiration from the Eclipse feature.
 
+## How to install the sublime-text plugin
+
+### OSX platform
+1. Open Sublime.
+2. Go to "Preferences -> Browse Packages".
+3. This would open a Finder with the package path. For me, it opens up this path: "/Users/ankitshubham/Library/Application Support/Sublime Text 2/Packages/".
```

Comment:
What about Sublime 3?

Reply:
It should work on Sublime-text 3 as well. This was just an example for the sake of clarity. I have Sublime-text 2 installed.

Reply:
Got it :)